chain/client/src/test_utils.rs

```diff
@@ -200,6 +203,105 @@ fn sample_binary(n: u64, k: u64) -> bool {
     thread_rng().gen_range(0, k) <= n
 }
 
+pub struct BlockStats {
+    hash2height: HashMap<CryptoHash, u64>,
+    num_blocks: u64,
+    max_chain_length: u64,
+    last_check: Instant,
+    max_divergence: u64,
+    last_hash: Option<CryptoHash>,
+    parent: HashMap<CryptoHash, CryptoHash>,
+}
+
+impl BlockStats {
+    fn new() -> BlockStats {
+        BlockStats {
+            hash2height: HashMap::new(),
```

Comment:
I would rename it to `hash2depth` or `hash2length` (and then consistently use `length` or `depth` throughout to refer to the number of blocks on the chain), to avoid the confusion with the actual block heights.

Reply:
It was marked as resolved, but the name is still `hash2height`. Not sure if there's a change that wasn't pushed yet?